### ISSUE TITLE
:bug: on Production: viewing Replay gives `Invalid record...

### ISSUE THREAD

**Issue by jperl** [RecordReplay/devtools#5576]
:bug: on Production: viewing Replay gives `Invalid recording ID` error

<https://www.qawolf.com/test/ckwgsrmhp000009mgeug4ftpi?suite_id=cky975er3000t09mp99g4bix6|View private recording> is showing this error but when navigating to this replay in incognito chrome, it shows the expected, `This is a private replay, Please sign in`
Also affecting <https://www.qawolf.com/test/cky4mkg4e000009jy28na7xa0|View team recording>

*Re-reporting to get into bug tracking*
Original bug report with discussion <https://qawolfhq.slack.com/archives/C02GEJCC9JP/p1641852950004800|here> \- [Chris](https://qawolfhq.slack.com/archives/C02GEJCC9JP/p1646246406974929)

**Comment by jaril** [comment]
@jperl I dug into this a week ago and forgot to leave a comment. I think what I found was that it never passed. Is there some timestamp on when it was last passing?

**Comment by jaril** [comment]
Apparently this started failing after https://github.com/RecordReplay/devtools/commit/916f79ed3f1b0eb6f5d3a90056c46e717f59107c

**Comment by jaril** [comment]
Asked QA Wolf to see if they can reproduce and send a Replay so we can debug the frontend piece.

Discovered that:
- We're getting the corresponding `CantViewRecording` logs in [Honeycomb](https://ui.honeycomb.io/replay/datasets/backend/result/zFkFFAXDNJA)

**Comment by jaril** [comment]
Replay link from this [Slack thread](https://replayio.slack.com/archives/C02GEJCC9JP/p1648502820502889)

https://app.replay.io/recording/private-recording-test--746fa070-c9dd-42b6-a36b-3184362e3c1e

**Comment by jaril** [comment]
<img width="902" alt="image" src="https://user-images.githubusercontent.com/15959269/161668010-e1ebb0dd-c84e-42e7-8760-79aac4e2141e.png">

https://replayio.slack.com/archives/C02GEJCC9JP/p1649111369964449

Notes: We found that there was a problem specifically with the way that authentication was working for QA Wolf that was leading to the bug. Refreshing the page would get the proper auth behavior. But we also found that splitting the two tests cases from one test run into two did the trick as well. QA Wolf proceeded with that second recommendation and now everything's passing! 🎉
